src/map.tsx

```diff
@@ -363,9 +370,28 @@ const ReactMapboxFactory = ({
         map.setStyle(nextProps.style);
       }
 
+      const handlerNames = ['doubleClickZoom', 'dragPan', 'dragRotate',
+        'touchZoomRotate', 'scrollZoom', 'boxZoom', 'keyboard']
+      for (const name of handlerNames) {
```

Comment:
Would you mind changing to forEach instead, so that it is consistent with the rest of the codebase?